docs/source/code_example/running_valkyrie_config.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+from xoa_core import (
+    controller,
+    types,
+)
+import asyncio
+import json
+from xoa_converter.entry import converter
+from xoa_converter.types import TestSuiteType
+
+PLUGIN_2544_PATH = "plugin2544"
+OLD_2544_CONFIG =  "old_2544_config.v2544" 
+PLUGIN_2889_PATH = "plugin2889"
+OLD_2889_CONFIG =  "old_2544_config.v2544" 
+
+async def convert_run_test(tester: types.Credentials, converter_type: TestSuiteType, plugin_path: str, suite_name: str, config):
+    # Create a default instance of the controller class.
+    ctrl = await controller.MainController()
+
+    # Register the plugins folder.
+    ctrl.register_lib(plugin_path)
```

Comment:
@ArtemConstantinov said we should
```
PLUGINS_PATH = Path(__file__).parent
...
ctrl.register_lib(PLUGINS_PATH)
```
it will load all plugins at once.

Reply:
OK, I'll update the doc